README.md

```diff
@@ -179,6 +179,30 @@ kumuluzee:
 When using both configuration keys and fully qualified class names for the configuration the fully qualified class
 name configuration takes precedence.
 
+### Configuration of underlying http client
+
+Corresponding jetty javadoc for jetty can be found [here](https://www.eclipse.org/jetty/javadoc/9.4.35.v20201120/org/eclipse/jetty/client/HttpClient.html).
+
+Configuration for `http-url` native java connector is not yet possible due to limitations in Jersey. 
+
+```yaml
+kumuluzee:
+  rest-client:
+    http-client-connector: jetty # or http-url, java native http client
+    enable-ssl-hostname-verification: true
+    disable-jetty-www-auth: true
```

Comment:
Should we deprecate `kumuluzee.rest-client.disable-jetty-www-auth` and move it to `kumuluzee.rest-client.jetty.disable-www-auth`? Probably the `enable-ssl-hostname-verification` too.

By that I mean start using new config keys in READMEs but still support old keys as a fallback for some time.